cli/kv_command.go

```diff
@@ -122,8 +125,11 @@ NOTE: This is an experimental feature.
 	watch.Arg("bucket", "The bucket to act on").Required().StringVar(&c.bucket)
 	watch.Arg("key", "The key to act on").Default(">").StringVar(&c.key)
 
-	ls := kv.Command("ls", "List available Buckets").Alias("list").Action(c.lsAction)
+	ls := kv.Command("ls", "List available buckets or the keys in a bucket").Alias("list").Action(c.lsAction)
+	ls.Arg("bucket", "The bucket to list the keys").StringVar(&c.bucket)
 	ls.Flag("names", "Show just the bucket names").Short('n').UnNegatableBoolVar(&c.listNames)
+	ls.Flag("verbose", "Show detailed info about the key").Short('v').BoolVar(&c.lsVerbose)
```

Comment:
can also be UnNegatableBoolVar, same with --display-value

Reply:
Sure thing. Will do.